colibre/registration.py

```diff
@@ -1690,6 +1690,42 @@ def register_stellar_birth_densities(self, catalogue):
     return
 
 
+def register_stellar_birth_temperatures(self, catalogue):
+
+    try:
+        average_log_T = catalogue.get_quantity("stellar_birth_temperatures.logaverage")
+        stellar_mass = catalogue.get_quantity("apertures.mass_star_100_kpc")
```

Comment:
Should this be 50kpc instead of 100, since that's what we're plotting? register_stellar_birth_densities would also need updated

Reply:
Indeed! I've fixed it.